nixos/modules/misc/ids.nix

```diff
@@ -355,6 +355,7 @@ in
       rstudio-server = 324;
       localtimed = 325;
       automatic-timezoned = 326;
+      shoko = 328;
```

Comment:
Please don't do that and just let it auto-allocate a UID from the regular pool (`isSystemUser=true` and no uid in actual user)